### ISSUE TITLE
How to disable pruning?

### ISSUE THREAD

**Issue by Legomountain14** [jekhokie/raspberry-noaa-v2#1076]
I would like an option to disable pruning, or a config value that can disable it in the docs.

Perhaps setting ```delete_oldest_n``` to 0?

If' it's already possible, it would be great for it to be documented.

**Comment by Mraanderson** [comment]
I agree that the pruning method and documentation could be better managed. the documentation is a bit vague and the cat example actually makes an entry in crontab. https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/pruning.md

You could set the days to keep to a very long time. Possibly having a lock button or star favourite on the passes that should be kept indefinitely so they are exempt from deletion. 

A delete immediately button would also be good for those you know you don't want straight away. (Edit: delete from the admin web control area for passes is suitable for the requirement and safer than having on the main page)
